### PR TITLE
Conformance tests for JavaScript (Node.js). 15 tests are failing.

### PR DESCRIPTION
**Pull request by haberman**
Review to @joeltine.

### CODE REVIEW

**Comment by haberman** [comment]
retest this please

File: conformance/conformance_nodejs.js [hunk 1]
```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env node
+
+/*
+ * Protocol Buffers - Google's data interchange format
+ * Copyright 2008 Google Inc.  All rights reserved.
```
Comment (joeltine) [inline conformance/conformance_nodejs.js]: 2016?

**Comment by joeltine** [comment]
@haberman high level comment: can you please provide more context on what this change is trying to accomplish? I'm going into this without any background and a summary would help. A top-level comment in the conformance test wouldn't be a bad idea.

File: conformance/conformance_nodejs.js [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+  } else {
+    throw "conformance_nodejs: premature EOF on stdin.";
+  }
+}
+
+// Utility function to read a buffer of N bytes.
+function readBuffer(bytes) {
+  var buf = new Buffer(bytes);
+  var totalRead = 0;
+  //console.warn("Want to read: " + bytes);
```
Comment (acozzette) [inline conformance/conformance_nodejs.js]: It might be good to delete these commented out `console.warn` statements.

**Comment by acozzette** [comment]
Could you perhaps add some short instructions to conformance/README.md about how to run the new tests? Otherwise LGTM!

**Comment by haberman** [comment]
Addressed your comments, PTAL. Thanks!

**Comment by acozzette** [comment]
LGTM

**Pull request merged**